database/mongodb/mongodb.go

```diff
@@ -184,10 +219,81 @@ func (m *Mongo) Drop() error {
 	return m.db.Drop(context.TODO())
 }
 
-func (m *Mongo) Lock() error {
+func (m *Mongo) ensureLockTable() error {
```

Comment:
We should chat about this function here and how safe it is.

Reply:
We should chat about this function here and how safe it is.